source/includes/job-board/_applications.md

```diff
@@ -147,6 +157,14 @@ email | Applicant's email adress
 *cover_letter | Cover letter file contents.  *Only allowed in `multipart/form-data` requests*
 *educations | An array of education objects. Each education object should have five fields: school_name_id, degree_id, discipline_id, start_date, and end_date. start_date and end_date will use a hash of month and year. You can get the school_name_id, degree_id, discipline_id from our [List Schools](#list-schools), [List Degrees](#list-degrees), and [List Disciplines](#list-disciplines) endpoints.
 
+### Submitting Attachments
+
+We support 3 ways to attach files to your payloads.
+
+1. Sending the document as an attachment in a multipart form.
+1. Sending URL and filename of the document.
+1. Sending a Base64 encoded string of the document and its filename.
```

Comment:
@jcmuller Should these say 2 and 3 instead of 1?

Reply:
Markdown numbered lists can be just `1.` or `a.`